client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html

```diff
@@ -36,14 +36,18 @@ <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"><
                     <div class="row my-4">
                         <div class="col-xs-12 col-md-12">
                             <label class="control-label"
-                                   for="simultaneousLines">
+                                   for="simultaneousLines"
+                                   data-oui-tooltip-placement="top-end"
+                                   data-oui-tooltip-append-to-body="true"
```

Comment:
According to the [`oui-tooltip`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/tooltip#api) documentation, there is no `append-to-body` attribut available. Could you please remove it?